src/views/Lottery/components/PreviousRoundCard/Footer.tsx

```diff
@@ -1,82 +1,38 @@
-import React, { useState } from 'react'
-import styled from 'styled-components'
-import { Flex, ExpandableLabel, CardFooter, Skeleton, Heading, Box, Text } from '@pancakeswap/uikit'
+import React, { useState, useEffect } from 'react'
+import { Flex, ExpandableLabel, CardFooter, Skeleton } from '@pancakeswap/uikit'
+import usePreviousValue from 'hooks/usePreviousValue'
 import { useTranslation } from 'contexts/Localization'
 import { LotteryRound } from 'state/types'
-import { usePriceCakeBusd } from 'state/farms/hooks'
-import { useGetLotteryGraphDataById } from 'state/lottery/hooks'
-import { formatNumber, getBalanceNumber } from 'utils/formatBalance'
-import Balance from 'components/Balance'
-import RewardBrackets from '../RewardBrackets'
+import FooterExpanded from './FooterExpanded'
 
-const NextDrawWrapper = styled(Flex)`
```

Comment:
Broke this component into two to make it easier to manage loading states - all deleted code is now in `FooterExpanded`